label/label.go

```diff
@@ -65,9 +65,7 @@ var (
 	labelRepoRegexp = regexp.MustCompile(`^@$|^[A-Za-z.-][A-Za-z0-9_.-]*$`)
 	labelPkgRegexp  = regexp.MustCompile(`^[A-Za-z0-9/._@-]*$`)
 	// This was taken from https://docs.bazel.build/versions/main/build-ref.html#name
-	// Note: We've manually removed space from the regex, because though these technically parse
-	// with Bazel (and can appear in query results), they cannot actually be used in practice.
```

Comment:
Can you explain why this comment is no longer true?

Reply:
It was never really true - I made a limiting assumption which was incorrect. https://github.com/bazelbuild/bazel/issues/4327 is an issue tracking fixing gaps in this support elsewhere in Bazel, too, which is an open and accepted issue, not something closed as working as intended.